lib/client/api.go

```diff
@@ -600,6 +600,11 @@ func readProfile(profileDir string, profileName string) (*ProfileStatus, error)
 		if err != nil {
 			return nil, trace.Wrap(err)
 		}
+		// If the cert expiration time is less than 5s consider cert as expired and don't add
+		// it to the user profile as an active database.
+		if time.Until(cert.NotAfter) < 5*time.Second {
+			continue
+		}
```

Comment:
This may be fine but seems a little "arbitrary" TBH. What's the reasoning for this? Ideally, the comment should explain not "what" but "why".